email/src/main/java/edu/jhuapl/dorset/demos/EmailClient.java

```diff
@@ -0,0 +1,111 @@
+package edu.jhuapl.dorset.demos;
+
+import java.util.Scanner;
+
+import javax.mail.Message;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+
+import edu.jhuapl.dorset.Application;
+import edu.jhuapl.dorset.Request;
+import edu.jhuapl.dorset.Response;
+import edu.jhuapl.dorset.agents.Agent;
+import edu.jhuapl.dorset.agents.DateTimeAgent;
+import edu.jhuapl.dorset.routing.Router;
+import edu.jhuapl.dorset.routing.SingleAgentRouter;
+
+public class EmailClient {
+
+    private static final Logger logger = LoggerFactory.getLogger(EmailClient.class);
+    private EmailManager manager;
+    private static Scanner scan;
+
+    /**
+     * EmailClient
+     *
+     * @param manager   EmailManager object
+     */
+    public EmailClient(EmailManager manager) {
+        this.manager = manager;
+    }
+
+    /**
+     * Main method in Email Client
+     * 
+     * @param args   command line arguments
+     */
+    public static void main(String[] args) {
+        scan = new Scanner(System.in);
```

Comment:
Move the scanner into the run method. Create it at the top of the run method and then close it at the end in a finally block. You don't use it anywhere else so better to have its creation where it is being used.